js/character-counter.js

```diff
@@ -6,6 +6,14 @@
 		this.characterCounter.call( el );
 	}
 
+	var length = function( e ){
+		if( /Safari|Chrome/.test(window.navigator.userAgent) ){
```

Comment:
@zachleat 

I tried creating a `textarea` programatically but the `input.value = val` assignment doesn't respect the maxlength. In which case we can't test how it respects maxlength. I'm open to other ideas if you have them.